lib/stm32/f4/rcc.c

```diff
@@ -307,6 +357,22 @@ const struct rcc_clock_scale rcc_hse_25mhz_3v3[RCC_CLOCK_3V3_END] = {
 		.apb1_frequency = 42000000,
 		.apb2_frequency = 84000000,
 	},
+    { /* 96MHz */
+		.pllm = 25,
+		.plln = 192,
```

Comment:
The motivation for 96MHz is that it's a fairly common frequency that
seems to be missing from the list. And yes, it should have been a
separate commit.

On 09/05/2016 11:54 AM, Karl Palsson wrote:
>> @@ -307,6 +357,22 @@ const struct rcc_clock_scale rcc_hse_25mhz_3v3[RCC_CLOCK_3V3_END] = {
>>  		.apb1_frequency = 42000000,
>>  		.apb2_frequency = 84000000,
>>  	},
>> +    { /* 96MHz */
>> +		.pllm = 25,
>> +		.plln = 192,
> 
> hang on, is this just more clock configs for 96mhz, mixed in with a commit that is for enabling convenience functions for HSI?  that's confusing.  Also, what's the particular motivation for 96mhz configs anyway?  Is that the max speed of any parts?  Custom configs like this can just be created in user code.
>